index.html

```diff
@@ -4,83 +4,219 @@
 </head>
 
 <body>
-	<div class="header">
-
-
-	</div><!-- /.container -->
-	<div class="main" style="background-image: url('images/grid.png')">
-		<div class="cover blue" data-color="blue"></div>
+	<div class="parallax"></div>
+	<div class="hero rounded shadow-lg">
+		<div class="container top">
+			<nav class="navbar sticky-top navbar-dark">
+				<div class="container">
+					<a class="navbar-brand logo_text" href="#">
+						<img src="./images/logo-circle.png" alt="" width="30" height="30" class="d-inline-block align-text-top">
+						Virtision
+					</a>
+				</div>
+			</nav>
+		</div>
 		<div class="container">
-
-			<div class="content">
+			<div class="row">
 				<div class="col-lg-6">
-					<img class="logo-img" src="./images/logo.png">
-					<h1 class="logo pop">
-						Virtision
-					</h1>
-					<h4 class="motto">Easily create, share and enjoy immersive worlds together</h4>
+					<h1 class="logo pop">The XR Social Music Platform</h1>
+					<h2 class="motto">Easily create, share and enjoy immersive worlds together</h2>
 					<div class="subscribe">
 						<h5 class="info-text">
 							Join the waiting list for the beta. We keep you posted.
 						</h5>
 						<div>
-							<form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
-								<div class="form-group">
-									<label class="sr-only" for="exampleInputEmail2">Email address</label>
-									<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
-									<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+							<form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+								target="_blank" novalidate>
+								<div class="row">
+									<div class="form-group">
+										<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
+										<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+									</div>
+									<button type="submit" class="btn btn-first btn-email btn-fill">NOTIFY ME</button>
 								</div>
-								<button type="submit" class="btn btn-email btn-fill">Notify Me</button>
-
 							</form>
-							<div class="media">
-								<a href="#">
+
+							<div class="media mb-3">
+								<a href="https://www.linkedin.com/company/virtision-com">
 									<i class="fa fa-linkedin-square"></i>
 								</a>
```

Comment:
<img width="164" alt="Screen Shot 2021-04-28 at 11 56 54 PM" src="https://user-images.githubusercontent.com/8165314/116504293-96d74200-a87d-11eb-9a5b-b6372feadd25.png">

Why's there these underscores after the LI and FB icons?

Reply:
yeah for some reason it's show like that, I will change the email icon to instagram icon instead

Reply:
weird the instagram isnt like that either

Reply:
oh text decoration fixed